charts/ocis/docs/values-desc-table.adoc

```diff
@@ -882,6 +882,12 @@ a| [subs=-attributes]
 a| [subs=-attributes]
 `"notifications-smtp-secret"`
 | Reference to an existing SMTP email server settings secret (see xref:{secrets}[Secrets]). Not used if `features.emailNotifications.enabled` equals `false`.
+| secretRefs.s3CredentialsSecretRef
+a| [subs=-attributes]
++string+
+a| [subs=-attributes]
+`"s3-credentials-secret"`
+| Reference to an existing s3 secret (see xref:{secrets}[Secrets]).Only used if storageusers.storage.storageBackend.driver is "s3ng".
```

Comment:
`(see xref:{secrets}[Secrets]).Only used if` there is a blank missing, this should be:
`(see xref:{secrets}[Secrets]). Only used if`

To be fixed in the source file